src/app/clients/page.tsx

```diff
@@ -9,16 +9,24 @@ const dataFetch: () => Promise<Schema["clients"][] | null> = async () => {
 };
 
 const Clients: () => Promise<React.ReactElement> = async () => {
-    const data = await dataFetch();
+    let data: Datum[] | null = await dataFetch();
+
+    if (data === null) {
+        data = [];
+    }
+
+    const headers: { [key: string]: string } = {
```

Comment:
this type is probably not needed :)
only functions and things which use custom interfaces need type annotations!

Reply:
Refactored as suggested